### PR TITLE
MTV 2.6.1 Release notes

### PR DESCRIPTION
**Pull request by RichardHoch**
MTV 2.6.1

Resolves https://issues.redhat.com/browse/MTV-1093 by updating the MTV release notes for version 2.6.1.

Preview: https://file.emea.redhat.com/rhoch/261_rn/html-single/#resolved-issues-26_release-notes [first three resolved issues].

### CODE REVIEW

**Comment by RichardHoch** [comment]
@ahadas Please review.

File: documentation/modules/rn-2.6.adoc [hunk 1]
```diff
@@ -178,6 +171,18 @@ For a complete list of all known issues in this release, see the list of link:ht
 
 This release has the following resolved issues:
 
+.VMs migrated from vSphere and OVa files are not being fully copied
```
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:174]:
```suggestion
.VMs migrated from vSphere and OVA files are not being fully copied
```

File: documentation/modules/rn-2.6.adoc [hunk 2]
```diff
@@ -178,6 +171,18 @@ For a complete list of all known issues in this release, see the list of link:ht
...
+
+In {project-short} 2.6.0, there was a problem copying VMs from VMware vSphere and from OVA files, resulting in the failure of migrations using these source providers. The problem was resolved in {project-short} 2.6.1. link:https://issues.redhat.com/browse/MTV-1067[(MTV-1067)]
+
+.Migrating VMs from one OCP-V cluster to another fails due to a timeout
+
+In {project-short} 2.6.0, migrations from one OCP-V cluster to another failed when the time to transfer the disks of a VM exceeded the timeout of the Export API in {ocp-name}, which was set to 2 hours by default. The problem was resolved in version 2.6.1 by setting the default value of the API to 12 hours, which greatly reduces the possibility of a timeout. Additionally, you can increase or decrease the the timeout as needed. link:https://issues.redhat.com/browse/MTV-1052[(MTV-1052)]
```
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:180]:
```suggestion
In {project-short} 2.6.0, migrations from one OCP-V cluster to another failed when the time to transfer the disks of a VM exceeded the timeout of the Export API in {ocp-name}, which was set to 2 hours by default. The problem was resolved in version 2.6.1 by setting the default value of the API to 12 hours, which greatly reduces the possibility of a timeout. Additionally, you can increase or decrease the timeout as needed. link:https://issues.redhat.com/browse/MTV-1052[(MTV-1052)]
```

File: documentation/modules/rn-2.6.adoc [hunk 3]
```diff
@@ -155,21 +155,14 @@ Deleting a migration plan does not remove temporary resources such as importer p
...
 
-.{project-short} forklift-controller pod crashes when receiving a disk without a datastore
-
-In some rare cases, the VM may be configured with a disk that is placed on a datastore that is no longer available in vSphere. MTV gets such disks without their datastore. Previously, the `forklift-controller` crashed in this situation, rendering {project-short} unusable. In {project-short} 2.6.0, {project-short} presents a critical validation for VMs with such disks, and the `forklift-controller` no longer crashes. link:https://issues.redhat.com/browse/MTV-1029[(MTV-1029)]
-
 .Failure to migrate an image-based VM from {osp} to the `default` project
 
 The migration process fails when migrating an image-based VM from {osp} to the `default` project. link:https://issues.redhat.com/browse/MTV-964[(MTV-964)]
+
 For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12430275[Known Issues] in Jira.
```
[the remark this answers is not in the document]
Comment (anarnold97) [reply documentation/modules/rn-2.6.adoc:166]: Please can you also update the *Resolved Issues* filter to

```
https://issues.redhat.com/issues/?filter=12435572
```

Thanks

**Comment by anarnold97** [comment]
A few little things, but apart from these LGTM

**Review by anarnold97 — APPROVED** [review]
LGTM, thanks

**Comment by RichardHoch** [comment]
Botched a rebase; opening new branch and PR just to be safe.

**Pull request closed without merging**